tidb-cloud/learning-path.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Learning Path
+hide_title: true
+hide_sidebar: true
+---
+
+<!--
+注意 frontmatter 里的内容
```

Comment:
Would you please translate all the Chinese text in this file to English? Let's use English in this repo. Thanks!